command_mode.c

```diff
@@ -727,7 +732,12 @@ static void cmd_showbind(char *arg)
 	if (*key == 0)
 		goto err;
 
-	/* FIXME: remove spaces at end */
+	/* remove trailing spaces from key */
+	char *key_end = key + strlen(key);
+	while (isspace(*(key_end-1))) {
+		key_end--;
+	}
+	*key_end = 0;
```

Comment:
Both addressed (will squash things later to clean up, if everyone is satisfied).

Just for my own education: what is wrong with `isspace()`?